### PR TITLE
Prefix items stored in localStorage with applicationId.

### PR DESCRIPTION
**Pull request by leonidr-da**
I think that this is probably good practice, but more purposefully, this will make it easier for us to build multiple demo's at the same time, and not have the odd login when one switches projects.

### CODE REVIEW

File: ui/src/config.ts [hunk 1]
```diff
@@ -16,11 +17,12 @@ export const httpBaseUrl = isLocalDev ? undefined : ('https://' + apiUrl.join('.
...
-
 export const createToken = (party : string) => jwt.sign({ "https://daml.com/ledger-api": { ledgerId, applicationId, admin: true, actAs: [party], readAs: [party] } }, "secret")
 
 let loginUrl = host.slice(1)
 loginUrl.unshift('login')
 
 export const dablLoginUrl = loginUrl.join('.') + (window.location.port ? ':' + window.location.port : '') + '/auth/login?ledgerId=' + ledgerId;
+
+export const daml_party_key = applicationId + ".daml.party";
+export const daml_token_key = applicationId + ".daml.token";
```
Comment (GeorgSchneider) [inline ui/src/config.ts:28]: Can you make these camelCase to be consistent?

Comment (leonidr-da) [reply ui/src/config.ts:28]: 👍

**Comment by leonidr-da** [comment]
@georg-da Back to you.

**Comment by leonidr-da** [comment]
@georg-da Can we please merge this ?

**Review by GeorgSchneider — APPROVED** [review]

**Comment by leonidr-da** [comment]
Thank you!

**Comment by GeorgSchneider** [comment]
Sorry for the tardiness with this one :/

**Pull request merged**